src/main/resources/db/migration/V11_4__alter_person_oversikt_status_add_columns_tilfelle.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE PERSON_OVERSIKT_STATUS
+ADD COLUMN tilfelle_latest              jsonb;
```

Comment:
Tanken er å lagre relevant informasjon fra Isoppfolgingstilfelle her, type:
tilfelle_start: Date
tilfelle_end: Date
tifelle_virksomhet: jsonb // {virksomhetsnavn, virksomhetsnummer}

og noen timestampz som kan brukes til oppdatering av tilfelle, i tillegg til i oppdatering av virksomhetsnavn, behandlende enhet, type:
tilfelle_updated_at: timestampz // til å 
tilfelle_bit_referanse_inntruffet: timestampz
tilfelle_bit_referanse_inntruffet: timestampz

Reply:
Hva er fordelen med å pakke disse inn i en json (dvs i stedet for å ha egne kolonner)?

Reply:
For å lagre en liste med virksomhetnumre så trengs det enten en egen relasjon eller evt en json for for:
`tifelle_virksomhet_listt: jsonb`
Om vi skal ha flat struktur med minst mulig jsonb i tabellen, så blir det fort en 5-6 kolonner for tilfelle-relaterte ting, hvorav en av evt blir jsonb. 
Alternativt man man nøste inn alt i en json for tilfelle, hva tenker du om disse to alternativene?